modules/video/src/optical_flow_io.cpp

```diff
@@ -76,14 +76,14 @@ CV_EXPORTS_W Mat readOpticalFlow( const String& path )
             if ( !file.good() )
             {
                 flow.release();
-                return std::move(flow);
+                return Mat();
             }
 
             flow(i, j) = u;
         }
     }
     file.close();
-    return std::move(flow);
+    return flow;
```

Comment:
Same as previous comment.
```suggestion
    return Mat(flow);
```

Reply:
Similar to the above, I also modified the code to use `Mat()`. thank you very much !